examples/example-gateway/build/gen-code/endpoints/multi/multi/versioncheck.go

```diff
@@ -0,0 +1,10 @@
+// Code generated by thriftrw v1.8.0. DO NOT EDIT.
```

Comment:
Your vendor packages (specifically thriftrw) are outdated, therefore the noisy diffs in this PR. Please reinstall the packages and then update the PR.